### PR TITLE
docs(observability): added how to retrieve your grafana credentials MTA-2253

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+meta: How to retrieve your Grafana credentials
+  title: How to retrieve your Grafana credentials
+  description: This page shows you how to retrieve your Grafana credentials
+content:
+  h1: How to retreieve your Grafana credentials
```
Comment (nedaehm) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:6]:
```suggestion
  h1: How to retrieve your Grafana credentials
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  paragraph: This page shows you how to retreieve your Grafana credentials
```
Comment (nedaehm) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:7]:
```suggestion
  paragraph: This page shows you how to retrieve your Grafana credentials
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  - observability
+dates:
+  validation: 2022-10-14
+  posted: 2022-10-03
+---
+
+This page shows you how to retrieve your Grafana credentials to access your Grafana dashboards (/observability/cockpit/concepts/#managed-dashboards) using the [Scaleway console](https://console.scaleway.com/).
+
+<Message type="requirement">
+  - You have an account and are logged into the [Elements Console](https://console.scaleway.com/)
```
Comment (bene2k1) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:18]:
```suggestion
  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  h1: How to retrieve your Grafana credentials
+  paragraph: This page shows you how to retrieve your Grafana credentials
+categories:
+  - observability
+dates:
+  validation: 2022-10-14
+  posted: 2022-10-03
+---
+
+This page shows you how to retrieve your Grafana credentials to access your Grafana dashboards (/observability/cockpit/concepts/#managed-dashboards) using the [Scaleway console](https://console.scaleway.com/).
```
Comment (RoRoJ) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:15]: missing the square brackets to designate link text

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  title: How to retrieve your Grafana credentials
+  description: This page shows you how to retrieve your Grafana credentials
+content:
+  h1: How to retrieve your Grafana credentials
+  paragraph: This page shows you how to retrieve your Grafana credentials
+categories:
+  - observability
+dates:
+  validation: 2022-10-14
+  posted: 2022-10-03
```
Comment (RoRoJ) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:12]: update

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+</Message>
+
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
+
+3. Click the **Grafana Users** tab. Your Grafana credentials display. YOu can copy them and [log in to your Grafana account](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).
```
Comment (RoRoJ) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:27]:
```suggestion
3. Click the **Grafana Users** tab. Your Grafana credentials display. You can copy them and [log in to your Grafana account](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).
```

**Review by bene2k1 — APPROVED** [review]

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  - observability
+dates:
+  validation: 2022-10-14
+  posted: 2022-10-03
+---
+
+This page shows you how to retrieve your Grafana credentials to access your Grafana [dashboards](/observability/cockpit/concepts/#managed-dashboards) using the [Scaleway console](https://console.scaleway.com/).
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
```
Comment (ofranc) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:18]:
```suggestion
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+</Message>
+
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
```
Comment (ofranc) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:23]:
```suggestion
1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```

**Review by ofranc — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+This page shows you how to retrieve your Grafana credentials to access your Grafana [dashboards](/observability/cockpit/concepts/#managed-dashboards) using the [Scaleway console](https://console.scaleway.com/).
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+</Message>
+
+
+1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```
Comment (nedaehm) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:23]:
```suggestion
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
```
Comment (nedaehm) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:25]:
```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+3. Click the **Grafana Users** tab. Your Grafana credentials display. You can copy them and [log in to your Grafana account](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).
```
Comment (nedaehm) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:27]:
```suggestion
2. Click the **Grafana Users** tab. Your Grafana credentials display. You can copy them and [log in to your Grafana account](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).
```

File: observability/cockpit/how-to/retrieve-grafana-credentials.mdx [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+This page shows you how to retrieve your Grafana credentials to access your Grafana [dashboards](/observability/cockpit/concepts/#managed-dashboards) using the [Scaleway console](https://console.scaleway.com/).
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+</Message>
+
+
+
+1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```
Comment (nedaehm) [inline observability/cockpit/how-to/retrieve-grafana-credentials.mdx:24]:
```suggestion
1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**